src/app/shared/cookies/browser-orejime.service.ts

```diff
@@ -106,13 +114,9 @@ export class BrowserKlaroService extends KlaroService {
    *  - Retrieves the current authenticated user
    *  - Checks if the translation service is ready
    *  - Initialize configuration for users
-   *  - Add and translate klaro configuration messages
+   *  - Add and translate orejime configuration messages
    */
   initialize() {
-    if (!environment.info.enablePrivacyStatement) {
-      delete this.klaroConfig.privacyPolicy;
-      this.klaroConfig.translations.zz.consentNotice.description = 'cookies.consent.content-notice.description.no-privacy';
```

Comment:
This seems to be removing the ability to *disable* the Privacy Statement in DSpace.  See the docs at https://wiki.lyrasis.org/display/DSDOC7x/User+Interface+Configuration#UserInterfaceConfiguration-Toggleend-useragreementandprivacypolicy

What happens if someone tries to use this PR when `enablePrivacyStatement: false` is set in your config.prod.yml?